pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -88,14 +79,14 @@ def solve_subproblem(solve_data, config):
                     MindtPy.variable_list,
                     solve_data.initial_var_values):
                 if not nlp_var.fixed and not nlp_var.is_binary():
-                    nlp_var.value = orig_val
+                    nlp_var.set_value(orig_val, skip_validation=True)
             # fixed_nlp.tmp_duals[c] = c_leq * max(
             #     0, c_leq*(value(c.body) - rhs))
             # TODO: change logic to c_leq based on benchmarking
```

Comment:
Do we still need this? This was a solved issue with the previous improvements to the code that can be removed.